tests/rest/admin/test_user.py

```diff
@@ -5500,3 +5500,125 @@ def test_redact_messages_all_rooms(self) -> None:
                     redaction_ids.add(event["redacts"])
 
         self.assertIncludes(redaction_ids, original_event_ids, exact=True)
+
+
+class GetInvitesFromUserTestCase(unittest.HomeserverTestCase):
+    servlets = [
+        synapse.rest.admin.register_servlets,
+        login.register_servlets,
+        admin.register_servlets,
+        room.register_servlets,
+    ]
+
+    def prepare(self, reactor: MemoryReactor, clock: Clock, hs: HomeServer) -> None:
+        self.admin = self.register_user("thomas", "pass", True)
+        self.admin_tok = self.login("thomas", "pass")
+
+        self.bad_user = self.register_user("teresa", "pass")
+        self.bad_user_tok = self.login("teresa", "pass")
+
+        self.random_users = []
+        for i in range(4):
+            self.random_users.append(self.register_user(f"user{i}", f"pass{i}"))
+
+        self.rm1 = self.helper.create_room_as(self.bad_user, tok=self.bad_user_tok)
+        self.rm2 = self.helper.create_room_as(self.bad_user, tok=self.bad_user_tok)
+        self.rm3 = self.helper.create_room_as(self.bad_user, tok=self.bad_user_tok)
```

Comment:
Nit: Prefer the full name instead of abbreviations.

```suggestion
        self.room3 = self.helper.create_room_as(self.bad_user, tok=self.bad_user_tok)
```